packages/expo-updates/ios/EXUpdates/UpdatesUtils.swift

```diff
@@ -111,6 +239,10 @@ public final class UpdatesUtils: NSObject {
     }
   }
 
+  internal static func sendStateEvent(_ type: UpdatesStateEventType, body: [String: Any] = [:]) {
+    AppController.sharedInstance.stateMachine.processEvent(UpdatesStateEvent(type: type, body: body))
```

Comment:
Same comment as android: we should separate out the concerns of these classes. Module/utils should call into controller, controller should hold business logic and update its state machine, and return back to the module/utils.